docker-compose.yaml

```diff
@@ -4,16 +4,19 @@ version: '3'
 
 services:
 
-  # For running the Flask server
-  web:
+  # For running the Flask server and tests
+  re_api:
     build:
       context: .
+      dockerfile: Dockerfile
       args:
         DEVELOPMENT: 1
     ports:
       - "127.0.0.1:5000:5000"
-    volumes:
-      - ${PWD}:/app
+#    uncomment to mount local directories
```

Comment:
Why is this commented?

Reply:
I had been using it during development but then commented it out whilst ensuring tests were passing, etc.